stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/view/FileAttachmentsView.kt

```diff
@@ -119,6 +119,18 @@ private class FileAttachmentViewHolder(
             fileTypeIcon.loadAttachmentThumb(attachment)
             fileTitle.text = attachment.getDisplayableName()
             fileSize.text = MediaStringUtil.convertFileSizeByteCount(attachment.fileSize.toLong())
+
+            // Todo: Use uploadState when possible
```

Comment:
Why is commented code here? Do you really need it?

Reply:
Because we need to use `uploadState` to display the correct state of the upload. `uploadState` is not sent by the Backend, but I talk with backend engineers and they told me a way to get the state of the upload, so I can uncomment this and program the correct behaviour. I'll change the PR to draft and work on this,